### PR TITLE
PSP_PIMS 16.0 EF scaffold.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs [hunk 1]
```diff
@@ -17,7 +17,6 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.SecurityDepositType, src => src.SecurityDepositTypeCodeNavigation.Description)
                 .Map(dest => dest.Description, src => src.Description)
                 .Map(dest => dest.AmountPaid, src => src.AmountPaid)
-                .Map(dest => dest.TotalAmount, src => src.TotalAmount)
```
Comment (devinleighsmith) [inline backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs:20]: @asanchezr this appears unused in deposits but want to confirm.

File: backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs [hunk 2]
```diff
@@ -20,8 +20,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.ReturnDate, src => src.ReturnDate)
                 .Map(dest => dest.ChequeNumber, src => src.ChequeNumber)
                 .Map(dest => dest.PayeeName, src => src.PayeeName)
-                .Map(dest => dest.PayeeAddress, src => src.PayeeAddress)
-                .Map(dest => dest.TerminationNote, src => src.TerminationNote);
```
Comment (devinleighsmith) [inline backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: @asanchezr same here.

File: backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs (hunk 1 above)
Comment (asanchezr) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs:20]: yes - this is no longer stored on DB. It is calculated from amount paid and interest to date

File: backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs (hunk 2 above)
Comment (asanchezr) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: * Payee Address should remain in the model/DB - not used in the UI (yet) but DB/model should still support it
* Termination Note was removed from row items and moved into parent Lease entity as TerminationNote I believe - can you confirm this?

**Review by FuriousLlama — APPROVED** [review]

File: backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs (hunk 2 above)
Comment (devinleighsmith) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: If we need payee address we'll need to get Doug to add it back in in a PSP_16_01 release. @dfilteau can you provide some background on why this was removed?
confirmed, termination note was moved to the lease.

Comment (asanchezr) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: I found it. It is called RETURN_NOTES on PIMS_LEASE table. TerminationNotes can be removed safely. Please keep Payee Address

Comment (asanchezr) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: @devinleighsmith I still see the field in the DB schema png
image

Comment (asanchezr) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: I would assume Doug's tool should not remove it....

Comment (devinleighsmith) [reply backend/api/Areas/Leases/Mapping/Lease/SecurityDepositReturnMap.cs:24]: Oh, no nvm, I'm being thick, its clearly still there sorry.

**Review by asanchezr — APPROVED** [review]

**Pull request merged**